### PR TITLE
Solution for classification task in module 1

### PR DESCRIPTION
**Pull request by terahekir**

### CODE REVIEW

**Comment by terahekir** [comment]
Hi @szikszail, please review

File: module-1/classification.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function grade(score) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+        gradeOfStudent = score < 0 || score > 100 ? 0 : Math.max(Math.min(Math.floor((score-40)/10),5),1);
```
Comment (szikszail) [inline module-1/classification.js:26]: Nice solution. Please ping me on Skype4Business for your badge.

**Review by szikszail — CHANGES_REQUESTED** [review]
If you'll add solutions for other tasks, I'd be happy to review them as well.

**Comment by szikszail** [comment]
Thank you for your solution!

**Pull request closed without merging**